app/scripts/controllers/apis/list.js

```diff
@@ -345,6 +377,7 @@ angular.module('apiExplorerApp').controller('ApisListCtrl', function($rootScope,
             }).finally(function() {
                 setFilteredApis();
                 loadAPIGroupOverview();
+				loadAPIStatusCode();
```

Comment:
Please fix the whitespace.  Do not use tabs, use spaces and make sure it lines up with existing code.